app/views/govuk_publishing_components/components/docs/title.yml

```diff
@@ -78,4 +78,15 @@ examples:
       margin_bottom: 0
     context:
       dark_background: true
-
+  using_design_system_template:
+    description: |
+      Page title component used in a [Design System page template](https://design-system.service.gov.uk/styles/page-template/default/index.html).
```

Comment:
Suggestion: `This option allows the removal of top margin from the component so that it works within a Design System page template, where spacing above the title is already provided by padding on the wrapping div.`